gammapy/maps/tests/test_base.py

```diff
@@ -316,3 +316,49 @@ def test_map_reproject_hpx_to_wcs():
     m_r = m.reproject(geom_wcs)
     actual = m_r.get_by_coord({"lon": 0, "lat": 0, "energy": [1.0, 3.16227766, 10.0]})
     assert_allclose(actual, [287.5, 1055.5, 1823.5], rtol=1e-3)
+
+
+map_arithmetics_args = [
```

Comment:
Arithmetic doesn't do anytihng with binsz, width, center and axes.
Suggest to just pick one map, or if you like one WCS and one HPX.
But even there: what's the point in having both?

Reply:
Well I took them for future geometry equality testing. Probably not needed since this tested already.